README.md

```diff
@@ -15,12 +15,18 @@ Registrator is a GitHub app that automates creation of registration pull request
 
 First, install the app on your package(s) as mentioned above.  The procedure for registering a new package is the same as for releasing a new version.
 
+1. Set the [Project.toml](Project.toml) version field in your repository to your new desired `version`.
```

Comment:
Maybe mention https://github.com/JuliaLang/Pkg.jl/blob/master/bin/gen_project.jl which can convert `REQUIRE` files to project files with (mostly) equivalent compat sections.

Reply:
I added the information in another section below